websites/S/Steam/presence.ts

```diff
@@ -23,314 +19,471 @@ presence.on("UpdateData", async () => {
+				else if (document.location.search.includes("tab=recent"))
+					presenceData.state = `Viewing ${pfname}'s recently played games`;
+				else if (document.location.pathname.includes("/awards"))
+					presenceData.state = `Viewing ${pfname}'s profile awards`;
+				else if (document.location.search.includes("tab=perfect"))
+					presenceData.state = `Viewing ${pfname}'s perfected games`;
+				else if (document.location.search.includes("tab=all"))
+					presenceData.state = `Viewing ${pfname}'s games`;
+				else if (document.location.pathname.includes("/inventory"))
+					presenceData.state = `Viewing ${pfname}'s inventory`;
+				else if (document.location.pathname.includes("/recommended"))
+					presenceData.state = `Viewing ${pfname}'s reviews`;
+				else presenceData.state = `Viewing ${pfname}'s profile details`;
+			} else if (
+				document.location.pathname.includes("/screenshots") ||
+				document.location.pathname.includes("/images") ||
+				document.location.pathname.includes("/videos") ||
+				document.location.pathname.includes("/myworkshopfiles")
+			) {
+				pfname = document.querySelector("#HeaderUserInfoName > a").textContent;
+				if (document.location.pathname.includes("/screenshots"))
+					presenceData.state = `Viewing ${pfname}'s screenshots`;
+				else if (document.location.pathname.includes("/images"))
+					presenceData.state = `Viewing ${pfname}'s artworks`;
+				else if (document.location.pathname.includes("/videos"))
+					presenceData.state = `Viewing ${pfname}'s videos`;
+				else if (document.location.search.includes("section=guides"))
+					presenceData.state = `Viewing ${pfname}'s guides`;
+				else if (document.location.search.includes("section=collections"))
+					presenceData.state = `Viewing ${pfname}'s workshop collections`;
+				else if (document.location.search.includes("section=merchandise"))
+					presenceData.state = `Viewing ${pfname}'s merchandise`;
+				else presenceData.state = `Viewing ${pfname}'s workshop items`;
+			} else if (
+				document.location.pathname.includes("/friends") ||
+				document.location.pathname.includes("/groups")
+			) {
+				pfname = document.querySelector("head > title").textContent;
+				if (document.location.pathname.includes("/friends")) {
+					if (document.location.pathname.includes("/common"))
+						presenceData.state = `Viewing ${pfname}'s friends in common`;
+					else presenceData.state = `Viewing ${pfname}'s friends list`;
+				} else if (document.location.pathname.includes("/groups")) {
+					if (document.location.pathname.includes("/common"))
+						presenceData.state = `Viewing ${pfname}'s groups in common`;
+					else presenceData.state = `Viewing ${pfname}'s joined groups`;
+				} else presenceData.state = `Viewing ${pfname}'s social details`;
+			} else if (document.location.pathname.includes("/home"))
+				presenceData.state = "Browsing the activity page";
+			else {
+				presenceData.smallImageKey = "lg";
```

Comment:
Imgur links are now preferred over Discord assets. Quick reminder: Assets must be 512x512. Also I highly suggest use an enum to store all of the assets.

Reply:
Again, as I said earlier (https://github.com/PreMiD/Presences/pull/7093#discussion_r1057840162), I'll leave this up to the original creator (@TheDropX )